.github/actions/bouf/action.yaml

```diff
@@ -0,0 +1,122 @@
+name: Run bouf
+description: Generates signed OBS install files and updater files
+inputs:
+  updaterPrivateKey:
+    description: Private key for signing update manifest
+    required: true
+  gcpWorkloadIdentityProvider:
+    description: GCP Identity Provider Pool ID
+    required: true
+  serviceAccountName:
+    description: Google service account name
+    required: true
+  version:
+    description: Version string (e.g. 30.0.0-rc1)
+    required: true
+  channel:
+    description: Update channel
+    required: false
+    default: 'stable'
+  commitHash:
+    description: Commit hash
+    required: false
+    default: ''
+
+runs:
+  using: composite
+  steps:
+    - name: Prepare Artifact
+      shell: pwsh
+      run: |
+        # Extract artifact in build dir and remove zip
+        cd build
+        Expand-Archive -Path *.zip -DestinationPath .
+        del *.zip
+
+    - name: Setup bouf
+      shell: pwsh
+      env:
+        BOUF_VERSION: 'v0.6.0'
+        BOUF_HASH: '06861ae6d714b754602000bd51607b5b72e655b978b7e7497b0eb970b1613dd3'
+        BOUF_NSIS_HASH: 'bb0561a76cecaba62785642dd71052a9fe53458e45df108ca08867ee333b80bd'
+      run: |
+        # Download bouf release
+        mkdir bouf && cd bouf
```

Comment:
Copy the `Ensure-Location` PowerShell module from the GitHub build scripts to the GitHub action and source it from there.

Reply:
The repo is checked out in this case so I can just import it rather than having to copy it, no? I don't think they'd be imported automatically.

Reply:
Yeah both work, I’d just prefer having them contained as that would make extracting them into a separate repo easier if we chose to do that.